content/blog/2017-11-28-fragments-in-react-v16.2.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Fragments in React v16.2"
+author: [clemmy]
+---
+
+Today, we're excited to improved support for fragments in React! We are introducing a new `Fragment` export, which allows
+which allows users to render *static* fragments using familiar JSX element syntax.
+
+```jsx
+import React, { Fragment } from 'react';
+
+render() {
+  return (
+    <Fragment>
+      <li>Look ma</li>
+      <li>a list!</li>
+    </Fragment>
+  );
+}
+```
+
+We've also included a new short syntax that is already supported by [some tools](#tooling-support):
+
+```jsx
+render() {
+  return (
+    <>
+      <li>Look ma</li>
+      <li>a list!</li>
+    </>
+  );
+}
+```
+
+### Why Fragments?
+
+React 16 added the [ability to return an array of elements from a component's `render` method](https://reactjs.org/blog/2017/09/26/react-v16.0.html#new-render-return-types-fragments-and-strings). This allowed developers to prevent extraneous markup with `div`s that pollute the DOM. However, it was inconvenient for users who were writing static JSX because the syntax is completely different from how they would usually write JSX:
+
+```jsx
+render() {
+  // keys and commas are needed when returning an array in a render
+  return [
+    <span key="first-span">I am</span>,
+    <span key="second-span">static</span>
+  ];
+}
+```
+
+So, we added the `Fragment` component to React! `Fragment`s let developers return multiple elements from `render` methods using a familiar syntax that resembles normal JSX elements:
+
+```jsx
+import React, { Fragment } from 'react';
+
+render() {
+  // keys and commas are no longer needed
+  return <Fragment>
```

Comment:
Should this be in a parenthesis with this line on the second line so that the comparison to the syntax below is equivalent?

Reply:
Yes, I'll update all the code blocks to have consistent styling with parentheses around JSX elements.